main.go

```diff
@@ -84,7 +86,12 @@ func setupCRDControllers(ctx context.Context, c client.Client, mgr ctrl.Manager)
 	clusterTypeProvider, err := clustertype.NewProvider(ctx, c)
 
 	cniBinDir := os.Getenv("CNI_BIN_DIR")
-	staticInfoProvider := staticconfig.NewProvider(staticconfig.StaticConfig{CniBinDirectory: cniBinDir})
+	useDTK := true
+	if os.Getenv("USE_DTK") == "false" {
```

Comment:
can you use config pkg for this ? 

see:
https://github.com/Mellanox/network-operator/blob/a592eb488955cf2e5678a4f6565459f579d7e638/pkg/config/config.go#L56

Reply:
Done